tools/dev_cluster.py

```diff
@@ -68,6 +68,8 @@ class RedpandaConfig:
     cloud_storage_disable_tls: bool = True
     cloud_storage_backend: str = 'aws'
     iceberg_enabled: bool = False
+    enable_developmental_unrecoverable_data_corrupting_features: int = int(
```

Comment:
Seems like a bummer we can't also set `development_enable_cloud_topics` as well

Reply:
it's not impossible--we'd just need to fiddle with the way config options are applied since `development_enable_cloud_topics` depends on `enable_developmental_unrecoverable_data_corrupting_features`.